### PR TITLE
[Enhancement] Remove the unnecessary thrift config

### PR DESCRIPTION
**Pull request by gengjun-git**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [x] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

Remove the thrift config thrift_rpc_timeout_ms and thrift_rpc_retry_times.

### CODE REVIEW

**Review by Astralidea — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:disappointed: **fail** : 1 / 5 (20.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/rpc/FrontendServiceProxy.java|0|2|00.00%|
|:large_blue_circle:|com/starrocks/qe/MasterOpExecutor.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/external/starrocks/TableMetaSyncer.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/server/NodeMgr.java|1|1|100.00%|

File: fe/fe-core/src/main/java/com/starrocks/external/starrocks/TableMetaSyncer.java [hunk 1]
```diff
@@ -30,9 +29,7 @@ public void syncTable(ExternalOlapTable table) {
         authInfo.setPasswd(table.getSourceTablePassword());
         request.setAuth_info(authInfo);
         try {
-            TGetTableMetaResponse response = FrontendServiceProxy.call(addr,
-                    Config.thrift_rpc_timeout_ms,
-                    Config.thrift_rpc_retry_times,
+            TGetTableMetaResponse response = FrontendServiceProxy.call(addr, 10000,
```
Comment (imay) [inline fe/fe-core/src/main/java/com/starrocks/external/starrocks/TableMetaSyncer.java:32]: why use a const value rather than a configure?

**Comment by padmejin** [comment]
Why do we remove this configuration? We should keep this configuration. After all, it's always easier to modify a configuration and restart than to change source code and rebuild.

**Comment by gengjun-git** [comment]
@chaoyli PTAL

**Pull request closed without merging**